core/client/src/main/java/alluxio/client/file/FileOutStream.java

```diff
@@ -84,27 +84,13 @@
   /**
    * Creates a new file output stream.
    *
-   * @param context the file system context
-   * @param path the file path
-   * @param options the client options
-   * @throws IOException if an I/O error occurs
-   */
-  public FileOutStream(FileSystemContext context, AlluxioURI path, OutStreamOptions options)
-      throws IOException {
-    this(path, options, context, UnderFileSystemFileOutStream.Factory.get());
-  }
```

Comment:
Not sure whether it is kept for backward compatibility.   we can make it package private.